src/SpectatorView.Unity/Assets/SpectatorView/Scripts/UI/NetworkConfigurationSettings.cs

```diff
@@ -0,0 +1,28 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using UnityEngine;
+
+namespace Microsoft.MixedReality.SpectatorView
+{
+    public class NetworkConfigurationSettings : Singleton<NetworkConfigurationSettings>
+    {
+        /// <summary>
+        /// Check to enable a mobile network configuration visual to obtain the user IP Address.
+        /// </summary>
+        [Tooltip("Check to enable a mobile network configuration visual to obtain the user IP Address.")]
+        [SerializeField]
+        private bool enableMobileNetworkConfigurationVisual = true;
+
+        /// <summary>
+        /// Prefab for creating a mobile network configuration visual.
+        /// </summary>
+        [Tooltip("Prefab for creating amobile network configuration visual, which replaces the defaultMobileNetworkConfigurationVisualPrefab on the SpectatorView component if set.")]
+        [SerializeField]
+        private GameObject overrideMobileNetworkConfigurationVisualPrefab = null;
+
+        public bool EnableMobileNetworkConfigurationVisual => enableMobileNetworkConfigurationVisual;
```

Comment:
We should really put the comments on the public properties instead of private fields. Generally we don't comment privates, and only add tooltip to private serializefields